lib/src/main/java/io/ably/lib/transport/ITransport.java

```diff
@@ -51,6 +51,7 @@ public TransportParams() {
 
 		public Param[] getConnectParams(Param[] baseParams) {
 			List<Param> paramList = new ArrayList<Param>(Arrays.asList(baseParams));
+			paramList.add(new Param("v", HttpUtils.X_ABLY_VERSION_VALUE));
```

Comment:
What do you want to test here? There won't be reauth messages from server if I remove this line.